swarm/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.4
+WORKDIR /go/src/github.com/docker
+RUN git clone https://github.com/smothiki/swarm
```

Comment:
Good catch--will fix. We will also change the branch name to something more appealing than "fail" (which means "failover" in this case).